src/games/strategy/net/DesktopUtilityBrowserLauncher.java

```diff
@@ -79,18 +83,19 @@ public static void openFile(final File file) {
    *        A web address (URL) of a web page (ex: "http://www.google.com/")
    */
   public static void openURL(final String url) {
-    try { // attempt to use Desktop library from JDK 1.6+
-      final Class<?> d = Class.forName("java.awt.Desktop");
-      d.getDeclaredMethod("browse", URI.class)
-          .invoke(d.getDeclaredMethod("getDesktop").invoke(null), URI.create(url));
-      // above code mimicks: java.awt.Desktop.getDesktop().browse()
-    } catch (final Exception ignore) { // library not available or failed
+    boolean failed = true;// TODO check if the failed thing is necessary
+    if (Desktop.isDesktopSupported()) {
+      try {
+        Desktop.getDesktop().browse(URI.create(url));
```

Comment:
As you may noticed, I have anothe PR in queue which removes most of the content of this class... Although it needs some testing it's very likely to replace this changes on merge...